nimble/core/data/base.py

```diff
@@ -4577,8 +4577,11 @@ def merge(self, other, point='strict', feature='union', onFeature=None,
                                                     onFeature, force)
         else:
             self._genericMergeFrontend(other, point, feature, onFeature)
-
-
+        
+        if onFeature:
+            self.points.sort(by=onFeature)
```

Comment:
Since the sorting is being done as part of accomplishing the merge, there's no need to log the sort individually. This is generally how helper calls to top level methods are handled all over the implementations; see the calls to `setNames` in `_genericMergeFrontend`, or the use of `copy` for `trainAndTestSets`. So the modified line would be
```
self.points.sort(by=onFeature, useLog=False)
```
which should eliminate the logging-related testing errors.

Reply:
*facepalm* Sorted on that front, ironically.